packages/docusaurus-mdx-loader/src/loader.ts

```diff
@@ -26,6 +26,7 @@ import type {LoaderContext} from 'webpack';
 // TODO as of April 2023, no way to import/re-export this ESM type easily :/
 // This might change soon, likely after TS 5.2
 // See https://github.com/microsoft/TypeScript/issues/49721#issuecomment-1517839391
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
https://typescript-eslint.io/rules/no-explicit-any was promoted to the `recommended` config in v6 (https://github.com/typescript-eslint/typescript-eslint/discussions/6014).

Reply:
Could you turn it off instead? I don't think it's quite useful for us, given our generally high quality types.